YANG/ccamp/Client-signal-yang/ietf-eth-tran-types.yang

```diff
@@ -314,32 +315,53 @@ module ietf-eth-tran-types {
         "Coupling Flag.";
     }
   }
-  
+
   identity topology-role {
-    description 
-    "The rold of underlay topology, e.g., hub, spoke, any-to-any. ";
+    description
+    "The rold of underlay topology, e.g.,
+    hub, spoke, any-to-any. ";
```

Comment:
Technical comment: it is not clear from the description and location within the tree whether this attribute represents the topology of the underlay tunnels (full-mesh or hub-and-spoke) or the role of the service end-point (any-to-any, hub/root or spoke/leaf)

In the former case (topology of the underlay tunnels), this attribute should be moved under the underlay container. More discussion is also needed since the list of tunnels should anyhow be explicitly configured and more complex topologies could also be required.

In the latter case (role of the service end-point), more discussion is needed since the configuration of the split-horizon groups could also be used to configure E-Tree services.

In both cases:
- the description needs further clarification;
- the option to define a enum instead of an identityref should be considered